src/main/java/sirius/db/mixing/Property.java

```diff
@@ -129,9 +139,10 @@ protected Property(@Nonnull EntityDescriptor descriptor, @Nonnull AccessPath acc
         this.propertyKey = field.getDeclaringClass().getSimpleName() + "." + field.getName();
         this.alternativePropertyKey = "Model." + field.getName();
         this.field.setAccessible(true);
-        this.name = accessPath.prefix() + field.getName();
+        this.name = accessPath.qualify(field.getName());
```

Comment:
driveby

Reply:
nope, this uses the method defined above -> refactoring

Reply:
yeah buts in the getFullLabel commit, thats what i meant